### PR TITLE
[revision] for {f77b8f32cde35c601f1538b4413e3ad6842235f4}

### PR DESCRIPTION
**Pull request by dpsmith**
Fix up documentation with Secure Launch Specification and SLRT.

### CODE REVIEW

File: Documentation/security/launch-integrity/secure_launch_overview.rst [hunk 1]
```diff
@@ -111,13 +111,22 @@ the mainline kernel where the TPM driver would be available.
...
+Outlined here is summary of the boot flow for Secure Launch. A more detailed
+review of Secure Launch process can be found in the Secure Launch
+Specification, a link is located in the `Resources`_ section.
+
 Pre-launch: *Phase where the environment is prepared and configured to initiate the
-secure launch in the GRUB bootloader.*
+secure launch by the boot chain.*
 
- - Prepare the CPU and the TPM for the launch.
+ - The SLRT is initialized and dl_stub is place in memory.
```
Comment (rossphilipson) [inline Documentation/security/launch-integrity/secure_launch_overview.rst:121]: s/place/placed

File: Documentation/security/launch-integrity/secure_launch_overview.rst [hunk 2]
```diff
@@ -111,13 +111,22 @@ the mainline kernel where the TPM driver would be available.
...
  - Load the kernel, initrd and ACM [2]_ into memory.
  - Setup the TXT heap and page tables describing the MLE [1]_ per the
    specification.
+ - If non-UEFI platform, dl_stub is called.
+ - If UEFI platforms, SLRT registered with UEFI and efi-stub called.
+ - Upon completion, the efi-stub calls dl_stub instead of EBS.
```
Comment (rossphilipson) [inline Documentation/security/launch-integrity/secure_launch_overview.rst:127]: These are not correct. efi-stub calls EBS before jumping to dlstub so this needs a little rewording.

Comment (dpsmith) [reply Documentation/security/launch-integrity/secure_launch_overview.rst:127]: ack

File: Documentation/security/launch-integrity/secure_launch_overview.rst (hunk 1 above)
Comment (dpsmith) [reply Documentation/security/launch-integrity/secure_launch_overview.rst:121]: ack

**Comment by dpsmith** [comment]
Changes force pushed, diff available here,
https://github.com/TrenchBoot/linux/compare/61ff0c37af7fffa8bce4f74c3c4716cc0bc8fab7..9526cce7bf9a0284c222a0cf11c19af40c68b021

**Pull request merged**